ski_lint/cli.py

```diff
@@ -27,45 +39,78 @@ def get_args() -> Namespace:
         help="return code is `1`, when non-ASCII files are found",
     )
     ap.add_argument("filenames", nargs="+", metavar="FILENAME", help="path to the files to check")
-    ap.add_argument("--context-width", type=int, default=50, help="width of the context of the non-ASCII line")
+    ap.add_argument("--context-width", type=int, help="width of the context of the non-ASCII line")
+    ap.add_argument("--config-file", type=str, help="path to config file")
```

Comment:
```suggestion
    ap.add_argument("-w", "--context-width", type=int, help="width of the context of the non-ASCII line")
    ap.add_argument("-c", "--config-file", type=str, help="path to config file")
```
I beliewe we should add shortened flags which will make the CLI run easier